tests/graphics/fig1d_test.py

```diff
@@ -182,3 +184,12 @@ def test_ax():
     da = data_array(ndim=1)
     _ = Figure1d(input_node(da), ax=ax)
     assert len(ax.lines) > 0
+
+
+def test_save_to_disk():
+    da = data_array(ndim=1)
+    fig = Figure1d(input_node(da))
+    with tempfile.TemporaryDirectory() as path:
+        fname = os.path.join(path, 'plopp_fig1d.pdf')
```

Comment:
You could check for all filetypes that we claim to support, e.g., by parametrizing the test.

Reply:
nice suggestion